packages/editor/src/components/post-publish-button/index.js

```diff
@@ -147,15 +148,19 @@ export class PostPublishButton extends Component {
 		};
 
 		const buttonProps = {
-			'aria-disabled': isButtonDisabled && ! hasNonPostEntityChanges,
+			'aria-disabled':
+				isButtonDisabled &&
+				( ! hasNonPostEntityChanges || isSavingNonPostEntityChanges ),
```

Comment:
Do you think this should be part of how `isButtonDisabled` is computed. Also might be an inline comment explaining the logic would help?

Reply:
Yeah, I think that would make sense, since this is the only place that we're using `isButtonDisabled` (and `isToggleDisabled`) for (the event handler -- `onClickButton` -- aside, but that obviously needs the same condition as the `aria-disabled` field).

Reply:
61d0634b91